clients/client/src/main/java/org/projectnessie/client/http/HttpClient.java

```diff
@@ -58,9 +59,12 @@ class Builder {
     private SSLContext sslContext;
     private SSLParameters sslParameters;
     private int readTimeoutMillis =
-        Integer.parseInt(System.getProperty("sun.net.client.defaultReadTimeout", "25000"));
+        Integer.parseInt(
```

Comment:
I think we missed `System.getProperty` hence the tests are failing.

Reply:
`DEFAULT_READ_TIMEOUT_MILLIS` and `DEFAULT_CONNECT_TIMEOUT_MILLIS` can be a string.

Reply:
Maybe `getInteger`?